services/thermal_mgr/thermal_mgr.c

```diff
@@ -43,18 +45,33 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
-
+  xQueueSend(thermalMgrQueueHandle, event, (TickType_t) 0);
```

Comment:
Perform error checking such as checking that the event isn't null, the queue handle has been setup, the queue isn't full